trompace/queries/person.py

```diff
@@ -32,7 +23,20 @@ def query_person(identifier: str=None,title: str=None, contributor: str=None, cr
         publisher (optional): An entity responsible for making the resource available
         honorific_prefix (optional): An honorific prefix.
         honorific_suffix (optional): An honorific suffix.
-        job_title (optional): The person's job title.
+        job_title (optional): The person's job title."""
+
+
+@docstring_interpolate("person_args", PERSON_ARGS_DOCS)
+def query_person(identifier: str=None,title: str=None, contributor: str=None, creator: str=None, source: str=None,
```

Comment:
you updated the fields in `query_mediaobject`, but didn't change any of them here. Do you think that all of these fields are necessary for queries?

Reply:
I've trimmed it down to a smaller list that I think is more appropriate for searching for a person.